symbols/arm9.yml

```diff
@@ -8670,7 +8670,7 @@ arm9:
         EU: 0xCC
         NA: 0xCC
         JP: 0xCC
-      description: "Note: unverified, ported from Irdkwia's notes"
+      description: Parameters used by the NitroSDK to read the ROM.
```

Comment:
It seems like we know the meanings of all the fields here, can we make a struct for it?

Reply:
I don't know enough about the struct to provide descriptions, but I can at least put the var names there.